### PR TITLE
docs: added pr template

### PR DESCRIPTION
**Pull request by reemardelarosa**
# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [x] Docs / tests

# Description

Added PR template.

# Use cases and why

1. To create uniformity to all Pull Requests submitted by contributors.

# Screenshots/Demo

Screenshot 2022-11-12 at 7 27 16 AM

# Checklist:

- [ ] My code follows the [developer guidelines of this project](https://github.com/reemardelarosa/qwik-ui/blob/main/CONTRIBUTING.md)
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

**Comment by reemardelarosa** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by reemardelarosa** [comment]
recheck

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- 1. One use case
+- 2. Another use case
+
+# Screenshots/Demo
+
+<!-- Add your screenshots here -->
+
+# Checklist:
+
+- [ ] My code follows the [developer guidelines of this project](https://github.com/reemardelarosa/qwik-ui/blob/main/CONTRIBUTING.md)
```
Comment (shairez) [inline .github/PULL_REQUEST_TEMPLATE.md:24]: the url should be updated to this repo

**Comment by shairez** [comment]
Thanks @reemardelarosa 
It seems to work now

I've added one small comment

**Pull request closed without merging**